youtube_dl_server/server.py

```diff
@@ -29,6 +29,14 @@ def main():
         help='The host the server will use. The default is: %(default)s',
     )
 
+    parser.add_argument(
+        '--number-processes',
+        default=5,
```

Comment:
I think it's not a good idea that the server does not respond to some requests immediately by default. If you leave  `1` by default and you request a video from "wait server" (like streamcloud) the server will not respond any requests for a while (12 seconds to streamcloud).

If `5` processes are many for you, reduce them to `2` or `3`; but I think `1` is a bad idea